cms/datavis/models/charts.py

```diff
@@ -97,8 +99,12 @@ def media(self) -> Media:
 
     def get_context(self, request: Optional["HttpRequest"] = None, **kwargs: Any) -> dict[str, Any]:
         config = self.get_component_config(self.primary_data_source.headers, self.primary_data_source.rows)
+        encoded_config = json.dumps(config, cls=DjangoJSONEncoder)
```

Comment:
I think that, because you (will be) using `json_script`, you don't need to encode this manually. As it'll likely end up with extra quotes

Reply:
The design system team explicitly wanted to receive the data as json rather than as a python object. But let me know how you would see it working.

Reply:
Do they want the macro argument to be a string, or just what's put on the page? If it's the latter, it's still fine to remove this.

Reply:
Yep let's make it a string and then do the json encoding with the custom filter as discussed.